main.go

```diff
@@ -30,11 +30,15 @@ import (
 
 var flagSet = flag.NewFlagSet("garble", flag.ContinueOnError)
 
-var garbleLiterals bool
+var (
+	garbleLiterals bool
+	debugDir       string
```

Comment:
probably a good time to start calling these `flagDebugDir` and so on, so that we don't use the wrong one in the code.